clippy_lints/src/lib.rs

```diff
@@ -776,7 +777,7 @@ pub fn register_plugins(store: &mut rustc_lint::LintStore, sess: &Session, conf:
             upper_case_acronyms_aggressive,
         ))
     });
-    store.register_late_pass(|| Box::new(default::Default::default()));
+    store.register_late_pass(|| Box::<default::Default>::default());
```

Comment:
At first this looked rather odd, interesting lint module + struct name we have there 😄

Reply:
Why, it's the default box for the Default lint in the default module. Totally clear, innit? :smile: